### PR TITLE
:sparkles: Add fork workflow functionality

### PR DESCRIPTION
**Pull request by xanjohns**
Adds the option of a "fork and pull request" workflow.

Can be enabled by specifying which user/bot account contains the forks, using the `FORK` input, defaulted to false.

- Changes to the cloned repository are pushed to the fork, instead of back upstream
- Pull requests' `head` is changed to the new branch on the forked repository

The advantage of this style is no write access is required to upstream repositories. This also means, however, that adding labels/assignees is not possible, so that has to be disabled if `FORK` is specified.

Note: This setup assumes the forks have already been created.

### CODE REVIEW

**Comment by xanjohns** [comment]
After having though about it a little more, I think it would make the most sense to include the step of actually forking repositories in this action as well, since we have already parsed the file containing the list of external repositories we are working with. I've added another commit to include the few lines needed to fork.

**Comment by BetaHuhn** [comment]
> After having though about it a little more, I think it would make the most sense to include the step of actually forking repositories in this action as well

Yeah, makes sense.

Still marked as a draft, is it ready to review?

**Comment by xanjohns** [comment]
Apologies for the slow response, but yes!

File: src/config.js [hunk 1]
```diff
@@ -108,6 +108,9 @@ try {
 		BRANCH_PREFIX: getInput({
 			key: 'BRANCH_PREFIX',
 			default: 'repo-sync/SOURCE_REPO_NAME'
+		}),
+		FORK: getInput({
+			key: 'FORK'
```
Comment (BetaHuhn) [inline src/config.js:113]: Please add a default value, in our case `false`.

You can also set the `FORK` option to `disableable` ([more info](https://github.com/BetaHuhn/action-input-parser#all-options)).

```suggestion
			key: 'FORK',
			default: false,
			disableable: true
```

**Review by BetaHuhn — CHANGES_REQUESTED** [review]

**Comment by BetaHuhn** [comment]
Other than the small change, everything looks good to me.

Could you add the `FORK` input to the end of the [Action Inputs table](https://github.com/BetaHuhn/repo-file-sync-action#%EF%B8%8F-action-inputs) and add an example on how to use it to the [examples section](https://github.com/BetaHuhn/repo-file-sync-action#-examples) in the README?

Will merge it after that.

**Review by BetaHuhn — APPROVED** [review]

**Comment by xanjohns** [comment]
I've added `FORK` to the Action Inputs table and provided an example/description.

**Comment by BetaHuhn** [comment]
Thanks!

**Pull request merged**